documentation/sphinx/source/api-ruby.rst

```diff
@@ -962,6 +969,9 @@ Asynchronous methods return one of the following subclasses of :class:`Future`:
 
         For a :class:`FutureNil` object returned by :meth:`Transaction.commit` or :meth:`Transaction.on_error`, you must call :meth:`FutureNil.wait`, which will return ``nil`` if the operation succeeds or raise an :exc:`FDB::Error` if an error occurred. Failure to call :meth:`FutureNil.wait` on a returned :class:`FutureNil` object means that any potential errors raised by the asynchronous operation that returned the object *will not be seen*, and represents a significant error in your code.
 
+.. class:: Int64Future
```

Comment:
Good catch on this one too. I think the way this was changed was that the `Version` class a few lines up became `Int64Future`, so probably the best way to do this doc change is to update the `Version` documentation with the new type name while also updating the couple of places in the documentation that refer to it.

Reply:
Updated. There is no reference to the `Version` class though.

Reply:
I think there are two usages from the `get_read_version` function, roughly at lines 522 and 731.

Reply:
Oh I see. I searched ':class:`Version`' and didn't find anything LOL.